bom-latest/pom.xml

```diff
@@ -6,7 +6,7 @@
         <artifactId>parent</artifactId>
         <version>${changelist}</version>
     </parent>
-    <artifactId>bom-2.303.x</artifactId>
+    <artifactId>bom-weekly</artifactId>
```

Comment:
Maybe rename the module to `bom-weekly` as well? If so, do not forget https://github.com/jenkinsci/bom/blob/7c549ea75075468aa38a00938b42cd8556c4e9ab/.github/dependabot.yml#L14